css/style.css

```diff
@@ -0,0 +1,199 @@
+/*................................................
+    HEADER SECTION
+.................................................*/
+
+.container {
+    max-width: 1600px;
+    min-width: 500px;
+    margin: auto;
+    z-index: 1;
+    background-color: white;
+    box-shadow: 0px 0px 30px #8a8585;
+    padding: 8px 30px;
+}
+
+.flex-box {
+    display: flex;
+    justify-content: space-between; 
+    list-style-type: none; 
+}
+
+header {
+    margin: 0 auto;
+}
+
+#logoImage {
+    max-width: 10%;
+}
+
+.flex-box a {
+    color: black;
+}
+.nav-bar {
+    margin-top: 55px;
+    vertical-align: middle;
+}
+
+nav li a:hover {
+    background-color: black;
+    color: white;
+}
+
+nav ul {
+    list-style-type: none;  
+}
+
+.flex-box li a {
+    display: flex;
+    padding-left: 20px;
+    padding-right: 20px;
+    font-size: 15px;
+    vertical-align: middle;
+    text-transform: uppercase;
+    text-decoration: none;
+    margin-bottom: 9px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: bold;
+}
+
+#icon {
+    padding: 15px;
+    font-size: larger;
+    margin-right: 100px;   
+}
+
+/*................................................
+    BODY SECTION
+.................................................*/
+body {
+    font-size: 16px;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+h2 {
+    font-size: 30px;
+}
+
+.content {
+    padding: 64px;
+    margin: 0 auto;
+    margin-top: 10px;
+    margin-bottom: 30px;
+    padding-top: 64px;
+    padding-bottom: 64px;
+    box-sizing: border-box;
+    background-color:#FFFF99 ;
```

Comment:
The color scheme chosen should match logo . Use  a color picker extension (example colorzilla) in chrome  to get exact color . Note that you will enable file URLs  in chrome settings. 

Also , easy way to get color scheme is to google color palettes with a desired color .
In this case , your logo is 'ochre' in color . Find a sample link [ochre-palette](https://cz.pinterest.com/pin/736268239062051361/)